docs/src/main/asciidoc/config.adoc

```diff
@@ -290,7 +290,7 @@ Setting the properties would occur in the normal manner, for example in `applica
 ----
 greeting.message = hello
 greeting.name = quarkus
-greeting.hidden.prizeAmount=10
+greeting.hidden.prize-amount=10
```

Comment:
@geoand Dominika had this issue while working on the doc. This has been like that since the beginning and the doc was incorrect or we changed the case at some point for the user properties?

Reply:
I don't remember us changing it, as far as I can remember, it was kebab case.